gnomad/utils/transcript_annotation.py

```diff
@@ -0,0 +1,92 @@
+"""Utils module containing generic functions that are useful for adding transcript expression-aware annotations."""
+import logging
+from typing import Callable, List, Optional, Tuple, Union
+
+import hail as hl
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("transcript_annotation_utils")
+logger.setLevel(logging.INFO)
+
+
+def summarize_transcript_expression(
+    mt: hl.MatrixTable,
+    transcript_expression_expr: Union[hl.expr.NumericExpression, str] = "x",
```

Comment:
```suggestion
    transcript_expression_expr: hl.expr.NumericExpression
```
`str` won't work.

Reply:
oh sorry, yes it will, I just forgot to change the code to use it (I meant to...)